lib/sovren-rest/client.rb

```diff
@@ -70,12 +70,26 @@ def headers
 
     ##
     # Builds up body of message for remote call.
-    def parse_body(input_file)
-      {
+    def parse_body(input_file, filemeta)
+      body = {
         'DocumentAsBase64String' => Base64.encode64(input_file),
         'OutputHtml' => 'true',
         'Configuration' => @configuration
       }
+
+      if filemeta.key?(:revision_date)
+        body['RevisionDate'] = format_timestamp(filemeta[:revision_date])
+      end
+
+      body
+    end
+
+    def format_timestamp(date)
+      return if date.nil? || !date.respond_to?(:strftime)
```

Comment:
Should this raise an exception instead if the provided value doesn't respond to `strftime`?